components/TxForm.tsx

```diff
@@ -123,16 +123,18 @@ export const TxForm = ({ recipientAddress }: { recipientAddress?: string }) => {
           <Flex justify="space-between">
             <FormLabel>Recipient address</FormLabel>
 
-            <Text
```

Comment:
(arbitray comment)

- I can't reproduce the width changing on my local or in my browser brave/firefox. But, I am seeing the height shift. Should I make the height larger?

Reply:
let's punt on it, we can adjust in another PR if necessary